src/main/resources/default.properties

```diff
@@ -0,0 +1,165 @@
+# app name
```

Comment:
@morungos why bake in the file? Why not just refer to it via something like a runtime parameter?